src/styles.css

```diff
@@ -198,6 +209,7 @@ table {
 }
 
 #waterChart {
+  background-color: white;
   color: #FFFFFF;
```

Comment:
We probably should have a different color than white for either the background or the text.